e3/anod/sandbox/action.py

```diff
@@ -195,6 +200,12 @@ def run(self, args):
 
             env = BaseEnv()
             cm = PlanContext(server=env)
+            resolver = AnodContext.always_create_source_resolver
+            if args.resolver:
+                if args.resolver in dir(AnodContext):
+                    resolver = getattr(AnodContext, args.resolver)
```

Comment:
remove the if args.resolver in dir(...) and use:

resolver = getattr(AnodContext, args.resolver, AnodContext.always_create_source_resolver)